config/module.config.php

```diff
@@ -1,44 +1,85 @@
 <?php
 
-return array(
-    'service_manager' => array(
-        'factories' => array(
-            'SlmQueueBeanstalkd\Options\BeanstalkdOptions' => 'SlmQueueBeanstalkd\Factory\BeanstalkdOptionsFactory',
-            'SlmQueueBeanstalkd\Service\PheanstalkService' => 'SlmQueueBeanstalkd\Factory\PheanstalkFactory',
-            'SlmQueueBeanstalkd\Worker\BeanstalkdWorker'   => 'SlmQueue\Factory\WorkerFactory'
-        )
-    ),
-
-    'controllers' => array(
-        'factories' => array(
-            'SlmQueueBeanstalkd\Controller\BeanstalkdWorkerController' => 'SlmQueueBeanstalkd\Factory\BeanstalkdWorkerControllerFactory',
-        ),
-    ),
+use SlmQueue\Strategy\MaxRunsStrategy;
+use SlmQueueBeanstalkd\Factory\BeanstalkdQueueFactory;
+use SlmQueue\Factory\WorkerFactory;
+use SlmQueueBeanstalkd\Controller\BeanstalkdWorkerController;
+use SlmQueueBeanstalkd\Factory\BeanstalkdWorkerControllerFactory;
+use SlmQueueBeanstalkd\Strategy\ClearObjectManagerStrategy;
+use SlmQueueBeanstalkd\Worker\BeanstalkdWorker;
 
-    'console'   => array(
-        'router' => array(
-            'routes' => array(
-                'slm-queue-beanstalked-worker' => array(
+return [
+    'service_manager' => [
+        'factories' => [
+            BeanstalkdWorker::class => WorkerFactory::class,
+            \Pheanstalk::class => \SlmQueueBeanstalkd\Factory\PheanstalkFactory::class
+        ]
+    ],
+    'controllers'     => [
+        'factories' => [
+            BeanstalkdWorkerController::class => BeanstalkdWorkerControllerFactory::class,
+            \Pheanstalk::class => \SlmQueueBeanstalkd\Factory\PheanstalkFactory::class
+        ],
+    ],
+    'console'         => [
+        'router' => [
+            'routes' => [
+                'slm-queue-Beanstalkd-worker'  => [
                     'type'    => 'Simple',
-                    'options' => array(
-                        'route'    => 'queue beanstalkd <queue> [--timeout=]',
-                        'defaults' => array(
-                            'controller' => 'SlmQueueBeanstalkd\Controller\BeanstalkdWorkerController',
+                    'options' => [
+                        'route'    => 'queue Beanstalkd <queue> [--timeout=] --start',
```

Comment:
Let's keep the lower case `b`. I think that is better, and it is identical to before.

Reply:
I find the way we use an argument to decide what action we are going to do perform a little bit strange. Let's remodel those to:

* `queue beanstalkd start <queue> [--timeout]`
* `queue beanstalkd recover <queue> `
* etc..